pkg/apis/openebs.io/v1alpha1/cstor_volume.go

```diff
@@ -67,6 +70,9 @@ type CStorVolumeStatus struct {
 	// +patchMergeKey=type
 	// +patchStrategy=merge
 	Conditions []CStorVolumeCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type" protobuf:"bytes,4,rep,name=conditions"`
+	// KnownReplicas represents the replicas that target can trust to read the
+	// data
+	KnownReplicas map[string]uint64 `json:"knownReplicas,omitempty"`
```

Comment:
even though this is string, lets make it as struct like `ReplicaDetails` so that any schema changes later would be extensible